### PR TITLE
Release Notes for WebView2: Added 1.0.1108.44 (v98) & 1.0.1158-prerelease (v100)

### PR DESCRIPTION
**Pull request by imicy**
Release Notes for Webview2(Edge98)

**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1735#10110844

Writer/Editor pass: vchapel

### CODE REVIEW

**Review by feich-ms — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -35,6 +35,72 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 98.1108.44 or higher.
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  The [Navigation API](/microsoft-edge/webview2/reference/win32/icorewebview2navigationstartingeventargs2?view=webview2-1.0.1108.44&preserve-view=true) that provides ability to provide additional allowed frame ancestors.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:53]: Is this a duplicate of the below 'add_NavigationStarting' api for iFrames? I don't think we added any modfication to our non iframe navigation api? @ViktoriyaBryhider would know.

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -35,6 +35,72 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+Release Date: February 6, 2022
+
+[NuGet package for WebView2 SDK 1.0.1108.44](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1108.44)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 98.1108.44 or higher.
+
+### General
+
+#### Promotions
+
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:50]: @cagriyildirim - is there a reason http auth is not on here? It was promoted to stable in prerelease last release. Should be stable release this time around if I'm not mistaken?

File: microsoft-edge/webview2/release-notes.md (hunk 1 above)
Comment (vbryh-msft) [reply microsoft-edge/webview2/release-notes.md:53]: It is AdditionalAllowedFrameAncestors https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalnavigationstartingeventargs?view=webview2-1.0.1083-prerelease - does not look like dupe.

**Review by jasonstephen15 — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**